### PR TITLE
Cleanup config and deploy scripts after cloud migration

### PR DESCRIPTION
**Pull request by devincowan**
Resolves https://github.com/hydroshare/hydroshare/issues/5510

### Pull Request Checklist: 
- [ ] Positive Test Case Written by Dev

- [ ] Automated Testing

- [ ] Sufficient User and Developer Documentation

- [ ] Passing Jenkins Build

- [ ] Peer Code review and approval

### Positive Test Case
1. [Enter positive test case here]

### CODE REVIEW

File: .github/workflows/hydroshare-pull-requests.yml [hunk 1]
```diff
@@ -32,10 +32,6 @@ jobs:
       run: |
         printf "%s\n" c | ./local-dev-first-start-only.sh
 
-    - name: Start HydroShare
-      run: |
-        docker-compose -f local-dev.yml up -d
-        sleep 60
```
Comment (sblack-usu) [inline .github/workflows/hydroshare-pull-requests.yml:38]: Is this not needed? The last line of local-dev-first-start-only is `docker-compose -f local-dev.yml down` so HS isn't running.

File: local-dev-first-start-only.sh [hunk 2]
```diff
@@ -426,35 +371,54 @@ echo '##########################################################################
...
+while [ 1 -eq 1 ]
+do
+  if [ "$NODE_CONTAINER_RUNNING" == "" ]; then
+    break
+  fi
+  echo -n "."
+  sleep 1
+done
 
-docker-compose -f local-dev.yml down
```
Comment (sblack-usu) [inline local-dev-first-start-only.sh:453]: ahh, I see that you leave HS running now!

Comment (devincowan) [reply local-dev-first-start-only.sh:453]: yeah I thought it would save some time

**Review by sblack-usu — APPROVED** [review]

**Comment by devincowan** [comment]
@sblack-usu requesting re-approval, fixed conflicts

**Review by sblack-usu — APPROVED** [review]

**Pull request merged**